tests/TestFileUtilities.py

```diff
@@ -338,13 +338,6 @@ def symlink_helper(link_fn):
             self.assertNotExists(linkname)
             self.assertNotLExists(linkname)
 
-        windows_vista_or_newer = False
-        if 'nt' == os.name:
```

Comment:
@livingentity 
It looks like we need to keep the line `if 'nt' == os.name:` so the kernel32.dll part does not run on posix systems. Otherwise, changes look good

Reply:
Thanks, done.